learning/inference.py

```diff
@@ -12,6 +12,8 @@
 from fastai.callback.wandb import WandbCallback
 from fastai.vision.learner import load_learner
 from utils import y_from_filename  # noqa: F401 (needed for fastai load_learner)
+import ffmpeg
+import subprocess
```

Comment:
What are these here for?

Reply:
I was trying to automate the ffmpeg video/gif making process with the photos inference produces. I believe subprocess can enter script into the command prompt.

Reply:
Yes, we could do that. But I didn't see that code anywhere. Did I miss it?

Reply:
No I have yet to implement it.